docs/detections/alerts-view-details.asciidoc

```diff
@@ -6,111 +6,141 @@
+The Visualizations section in the right panel offers a glimpse of the processes that led up to the alert and occurred after it. Click **Visualizations** to display the following sub-sections:
 
-[role="screenshot"]
-image::images/enriched-host-user-rs.png[Host and user risk score subsections with risk scores, 600]
+* **Session view preview**: Shows a preview of <<session-view,session view>> data. Click **Session viewer preview** to open the **Session View** tab in Timeline. 
 
-Two types of user and host risk scores can appear in the Enriched data section:
+* **Analyzer preview**: Shows a preview of the <<visual-event-analyzer,visual analyzer graph>>. The preview displays up to three levels of the analyzed event's ancestors and up to three levels of the event's descendants and children. The ellipses symbol (**`...`**) indicates the event has more ancestors and descendants to examine. Click **Analyzer preview** to open the **Event Analyzer** tab in Timeline.
 
-* **Current user/host risk classification:** The current risk score of the user or host associated with the alert.
-* **Original user/host risk classification:** The first risk score that was calculated for the user or host associated with the alert.
-+
-The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores appear, showing how the risk changed.
+[discrete]
+[[insights-section]]
+== Insights 
+
+The Insights section in the right panel offers different perspectives from which you can assess the alert. 
+
+Click **Insights** to display the following sub-sections:
+
+* <<entities-section>>
+* <<threat-intelligence-section>> 
+* <<correlations-section>>
+* <<prevalence-section>>
 
 [discrete]
-[[alerts-enrich-ti]]
-==== Alerts enriched with threat intelligence
-Alerts can be enriched with contextually relevant threat intelligence that you can use for triaging and investigating alerts. If available, this information appears in the Enriched data section.
+[[entities-section]]
+=== Entities 
 
-When gathering threat intelligence data for an alert, {elastic-sec} queries the alert for indicator matches from the past 30 days. The query looks for the following fields:
+The Entities sub-section shows the user related to the alert and the host on which the alert was generated against. <<host-risk-score,Host>> and <<user-risk-score,user risk>> scores are also available with a https://www.elastic.co/pricing[Platinum subscription] or higher. 
+
+NOTE: User and host risk scores are technical preview features. 
+
+Click **Entities** to expand the **Entities** sub-tab in the left panel. The **Entities** sub-tab provides more information about the user and host. 
```

Comment:
I think we can add more to the entities section, similar to others. the `related users` and `related hosts` tables are new in 8.10 and they are available only in the new flyout. we should highlight what you wrote in the tooltips i.e. `these users log in to this host after this alert is triggered`

Reply:
Addressed in [f9cbf44](https://github.com/elastic/security-docs/pull/3816/commits/f9cbf44419d47058906aee164a3b490e76d9e75e).